client/my-sites/plugins-wpcom/plugin-types/premium-plugin.jsx

```diff
@@ -38,4 +42,16 @@ PremiumPlugin.propTypes = {
 	description: PropTypes.string.isRequired
 };
 
-export default PremiumPlugin;
+const trackClick = name => recordTracksEvent(
+	'calypso_plugin_wpcom_click',
```

Comment:
@gwwar I merged without the `cta_name` since I didn't think this was particularly a nudge. I think this is the place people come to _after_ being nudged maybe